<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GraphWindow</class>
 <widget class="QMainWindow" name="GraphWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="chartWidget" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>561</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="generateReportButton">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>40</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>年收支趋势</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CategoryExpenseReportButton">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>313</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>支出分布</string>
    </property>
   </widget>
   <widget class="QWidget" name="chartWidget_2" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>260</y>
      <width>451</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="CategoryIncomeReportButton">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>360</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>收入分布</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menupersonal_financial_manager">
    <property name="title">
     <string>personal financial manager 图表</string>
    </property>
   </widget>
   <addaction name="menupersonal_financial_manager"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
